### PR TITLE
Updating instructions for URDF repository 

### PR DESCRIPTION
**Pull request by vidurvij-Unity**
1. Adding images to be used in URDF repo front page
2. Adding instructions for using disable_collisions tag in URDF

### CODE REVIEW

File: tutorials/urdf_importer/urdf_appendix.md [hunk 1]
```diff
@@ -316,6 +316,17 @@ To address this predicament we have integrated another algorithm to create Conve
 
 ![](images/ConvexMeshComparison.png)
 
+## Disable Collision Support
+URDF defines the robot visually using Visual Meshes. To define the physical volume of a robot, Unity use collision meshes. They are used to define the physical volume of the links and help in calculating inertia of the link and  detecting the collisions between different physical objects. When a collider mesh is imported in Unity, it is decomposed into simpler shapes to form a concave hull. This is necessary in detecting collisions between two mesh colliders. The changed shape might intersect with each other creating a hindrance in robot movement. To remedy this we support ```disable collision``` tag in URDF. To add an exception for collision detection in Unity:
```
Comment (hyounesy) [inline tutorials/urdf_importer/urdf_appendix.md:320]: , Unity **"uses"** collision meshes.

Comment (hyounesy) [reply tutorials/urdf_importer/urdf_appendix.md:320]: calculating **"the"** inertia

File: tutorials/urdf_importer/urdf_appendix.md [hunk 2]
```diff
@@ -316,6 +316,17 @@ To address this predicament we have integrated another algorithm to create Conve
...
+URDF defines the robot visually using Visual Meshes. To define the physical volume of a robot, Unity use collision meshes. They are used to define the physical volume of the links and help in calculating inertia of the link and  detecting the collisions between different physical objects. When a collider mesh is imported in Unity, it is decomposed into simpler shapes to form a concave hull. This is necessary in detecting collisions between two mesh colliders. The changed shape might intersect with each other creating a hindrance in robot movement. To remedy this we support ```disable collision``` tag in URDF. To add an exception for collision detection in Unity: 
+
+1. Identify the links between which you want to ignore the collisions.
+2. Add a tag in the URDF file with the format 
+```XML
+ <disable_collision link1= <name_of_link_1> link2=<name_of_link_2>>
+  </disable_collision>
+```
+    The disable collision tag flags the links to the parser that need to be ignored. Value of link1 and link2 attribute are the names of the two links between which the collision need to be ignored. Make sure the names of the links are same as defined in the URDF file.
```
Comment (hyounesy) [inline tutorials/urdf_importer/urdf_appendix.md:328]: f

**Review by hyounesy — APPROVED** [review]

File: tutorials/urdf_importer/urdf_appendix.md [hunk 3]
```diff
@@ -316,6 +316,16 @@ To address this predicament we have integrated another algorithm to create Conve
...
+URDF defines the robot visually using Visual Meshes. To define the physical volume of a robot, Unity uses collision meshes. They are used to define the physical volume of the links and help in calculating the inertia of the links and  detecting the collisions between different physical objects. When a collider mesh is imported in Unity, it is decomposed into near convex shapes to form a concave hull. This is necessary in detecting collisions between two mesh colliders. The changed shape might intersect with each other creating a hindrance in robot movement. To remedy this, we support ```disable collision``` tag in URDF. To add an exception for collision detection in Unity: 
+
+1. Identify the links between which you want to ignore the collisions.
+2. Add a tag in the URDF file with the format 
+```XML
+ <disable_collision link1= <name_of_link_1> link2=<name_of_link_2>>
+  </disable_collision>
+```
+    The disable collision tag flags the links that need to be ignored to the URDF parser. Values of link1 and link2 attributes are the names of the two links between which the collision needs to be ignored. Make sure the names of the links match the names defined in the URDF file.
```
Comment (mpinol) [inline tutorials/urdf_importer/urdf_appendix.md:328]: Remove the indentation.

File: tutorials/urdf_importer/urdf_appendix.md [hunk 4]
```diff
@@ -316,6 +316,16 @@ To address this predicament we have integrated another algorithm to create Conve
 
 ![](images/ConvexMeshComparison.png)
 
+## Disable Collision Support
+URDF defines the robot visually using Visual Meshes. To define the physical volume of a robot, Unity uses collision meshes. They are used to define the physical volume of the links and help in calculating the inertia of the links and  detecting the collisions between different physical objects. When a collider mesh is imported in Unity, it is decomposed into near convex shapes to form a concave hull. This is necessary in detecting collisions between two mesh colliders. The changed shape might intersect with each other creating a hindrance in robot movement. To remedy this, we support ```disable collision``` tag in URDF. To add an exception for collision detection in Unity:
```
Comment (mpinol) [inline tutorials/urdf_importer/urdf_appendix.md:320]: The second and third sentences say similar things.

```
To define the physical volume of a robot, Unity uses collision meshes. They are used to define the physical volume of the links and help in calculating the inertia of the links and  detecting the collisions between different physical objects.
```

Could be changed to something like,

```
Unity uses collision meshes to define the physical volume of the robot which are used to calculate the inertia of links and detect collisions between different physical objects.
```

File: tutorials/urdf_importer/urdf_appendix.md [hunk 5]
```diff
@@ -316,6 +316,16 @@ To address this predicament we have integrated another algorithm to create Conve
 
 ![](images/ConvexMeshComparison.png)
 
+## Disable Collision Support
+URDF defines the robot visually using Visual Meshes and its inertial volume using collision meshes. Inertial meshes used to define the physical volume of the links and help in calculating the inertia of the links and  detecting the collisions between different physical objects. When a collider mesh is imported in Unity, it is decomposed into near convex shapes to form a concave hull. This is necessary in detecting collisions between two mesh colliders. The changed shape might intersect with each other creating a hindrance in robot movement. To remedy this, we support ```disable collision``` tag in URDF. To add an exception for collision detection in Unity:
```
Comment (LaurieCheers-unity) [inline tutorials/urdf_importer/urdf_appendix.md:320]: Article needed here - we support **a/the** ```disable collision``` tag.

File: tutorials/urdf_importer/urdf_appendix.md [hunk 6]
```diff
@@ -316,6 +316,16 @@ To address this predicament we have integrated another algorithm to create Conve
...
+1. Identify the links between which you want to ignore the collisions.
+2. Add a tag in the URDF file with the format 
+```XML
+ <disable_collision link1= <name_of_link_1> link2=<name_of_link_2>>
+  </disable_collision>
+```
+The disable collision tag flags the links that need to be ignored to the URDF parser. Values of link1 and link2 attributes are the names of the two links between which the collision needs to be ignored. Make sure the names of the links match the names defined in the URDF file.
 
-
+Note: You can also manually ignore collisions in Unity using [APIs](https://docs.unity3d.com/ScriptReference/Physics.IgnoreCollision.html).
```
Comment (LaurieCheers-unity) [inline tutorials/urdf_importer/urdf_appendix.md:330]: Would also be nice to link to our example file: https://github.com/Unity-Technologies/Unity-Robotics-Hub/blob/main/tutorials/pick_and_place/PickAndPlaceProject/Assets/URDF/niryo_one/niryo_one.urdf

**Review by mpinol — APPROVED** [review]

**Pull request merged**